scio-elasticsearch/src/main/java/com/spotify/scio/elasticsearch/ElasticsearchIO.java

```diff
@@ -0,0 +1,231 @@
+package com.spotify.scio.elasticsearch;
```

Comment:
@lmayumi all source files have it, here for example: https://github.com/spotify/scio/blob/master/scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/package.scala